src/sdk/modules/smartSessions/smartSessions.test.ts

```diff
@@ -126,6 +128,25 @@ describe("modules.smartSessions.dx", async () => {
 
     expect(installSuccess).toBe(true)
 
+    // Trust the mock attester. 
+    // We're running on a fork of base sepolia, where necessary modules are registered on the registry and mock attestations are done.
+    const trustAttestersHash = await usersNexusClient.sendTransaction({
```

Comment:
@joepegler 

we can do this in bootstrap data, there is already means for that. that will set attesters upon deployment
https://github.com/bcnmy/nexus/blob/04e88dc2cc95b981e9b011cec8a736e3d8b39869/contracts/utils/NexusBootstrap.sol#L38

another way would be limited to SS usage where we batch this with createSessions call